open_instruct/dataset_processor.py

```diff
@@ -532,49 +532,6 @@ def visualize_token(tokens: list[int], tokenizer: PreTrainedTokenizer):
     console.print(rich_text)
 
 
-class SimplePreferenceCollator:
```

Comment:
Why did the collator get moved away from the other collators?

Reply:
I would like to remove `dataset_processor.py` in the long run. Also mainly because it's using `INPUT_IDS_CHOSEN_KEY`, but DPO uses `CHOSEN_INPUT_IDS_KEY`. Let me know if you'd like me to revert the changes to put them together.

Reply:
sounds good, just checking!